pages/learn/manage/organizations.md

```diff
@@ -0,0 +1,97 @@
+
+<img alt="Organization Team Summary" src="/img/common/app/" width="100%">
+
+
+## Creating new applications under your organization
+
+With members invited to the organization, let's get started on creating new applications under an organization. Navigate to the `Organization Applications` tab in the sidebar to find the `Create application` button for creating a new application under your organization. Learn more about creating an application from the [getting started](getting-started) guide.
+
+<img alt="New Application under Organization" src="/img/common/app/" width="100%">
+
+### Moving existing applications to an organization
+
+Moving an existing application from your personal or any other organization to another organization is a common task that can be accomplished by [transferring ownership](transferring-ownership) of your application.
+
+<img alt="Transfer Ownership" src="/img/common/app/" width="100%">
+
+### Providing access to multiple applications
+
+With organizations, providing access to individual members or teams becomes a hassle-free process. There are mainly 3 application-level roles that can be assigned, [`Developer`](developer), [`Observer`](observer) or [`Operator`](operator). Administrator default to [`Owner`](owner) role to all applications under a organization and are the only ones that can provide access to applications. Administrators won't be listed in members page of an organization.  
+
+- **Providing access to members** - Once you are done inviting members, click the username you want to provide access to from the `Members` list. When the member page opens, click `Manage applications` to see all applications available under the org. From there, click the `Add applications to member` button to add the member to one or more applications available under the org with the desired role that you want to assign.
+
+<img alt="Provide Member Access to Applications" src="/img/common/app/" width="100%">
+
+- **Providing access to teams** - Once you are done creating a team, navigate to the team you want to provide access in the `Organization Teams` tab.  Next, click the `Manage applications` button on the team page to see all applications available under the org. From there, click the `Add applications to team` button to add the team to one or more applications available under the org with the desired roles that you want to assign. All members of that team will be provided access to the selected application with the specified role per application.
+
+<img alt="Provide Team Access to Applications" src="/img/common/app/" width="100%">
+
+
+## Users as an organization - Sharing your apps just became hassle-free
+
+With organizations being a crucial feature for collaboration, we wanted the same for individual users. Each user in balenaCloud has their own "personal" organization named after their username. This "personal organization" with all applications and devices under it are private until explicitly shared by the user.
```

Comment:
I believe we don't want to call it "personal" but rather "default" org.

Reply:
Yes, that's a discussion-worthy thing we need to take a decision on the blog as well, I will check in on that before making a decision.

Reply:
@shaunmulligan We will be going with "default" org everywhere.